### ISSUE TITLE
first run - unauthenticated git protocol on port 9418 is not longer supported

### ISSUE THREAD

**Issue by cboecking** [cheat/cheat#664]
Thanks for submitting a bug report. Please provide the following information:

**A description of the problem**
installed using:
sudo snap install cheat

Installed cheat on a new 20.04 system via snap. Ran 'cheat tar'. System responded with no config - create one now? I said yes. Below is what happened:

~$ cheat tar
A config file was not found. Would you like to create one now? [Y/n]:
Would you like to download the community cheatsheets? [Y/n]:
Cloning into '/home/idempiere-admin/snap/cheat/common/.config/cheat/cheatsheets/community'...
fatal: remote error:
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
failed to run installer: failed to clone cheatsheets: failed to clone cheatsheets: exit status 128

**cheat version info**
Please paste the output of `cheat -v` here.
cheat -v
4.2.3

**Comment by bernermic** [comment]
Hi @cboecking  thanks for your report. I had a look into it and fixed the issue with the snap's git config.

Can you check this pls.?

**Comment by bernermic** [comment]
@chrisallenlane  can you close this one?

**Comment by cboecking** [comment]
Closed and thank you!

**Comment by chrisallenlane** [comment]
Thanks, guys.

**Comment by cboecking** [comment]
Hi Micheal,

Confirmed fixed. Here is how I confirmed:

   - could not re-run 'cheat config' due to previous bad state.
   - sudo snap remove cheat
   - sudo snap install cheat
   - cheat tar (said yes to config and download - all worked as expected)
   - cheat tar (all worked as expected)

Thank you for your time and effort regarding this project!

Regards,


Chuck Boecking
512.850.6068 (office and cell)
***@***.***
www.ChuckBoecking.com
